lib/plugins/aws/deploy/compile/events/apiGateway/tests/resources.js

```diff
@@ -81,10 +81,10 @@ describe('#compileResources()', () => {
         'bar/-': 'ApiGatewayResourceBarDash',
         'foo/bar': 'ApiGatewayResourceFooBar',
         foo: 'ApiGatewayResourceFoo',
-        'bar/{id}/foobar': 'ApiGatewayResourceBarIdFoobar',
-        'bar/{id}': 'ApiGatewayResourceBarId',
-        'bar/{foo_id}/foobar': 'ApiGatewayResourceBarFooidFoobar',
-        'bar/{foo_id}': 'ApiGatewayResourceBarFooid',
+        'bar/{id}/foobar': 'ApiGatewayResourceBarIdvarFoobar',
+        'bar/{id}': 'ApiGatewayResourceBarIdvar',
```

Comment:
isnt it better to capitalize V? so it'd be `ApiGatewayResourceBarIdVar` instead of `ApiGatewayResourceBarIdvar`. We do this normalization everywhere. Better be consistent